mifosng-android/src/main/java/com/mifos/api/datamanager/DataManagerDocument.java

```diff
@@ -85,9 +85,9 @@ public DataManagerDocument(BaseApiManager baseApiManager) {
      * @param entityType Entity Type
      * @param entityId   Entity Id
      * @param documentId Document Id
-     * @return GenericResponse
+     * @return DocumentRelatedResponse
      */
-    public Observable<GenericResponse> removeDocument(String entityType, int entityId,
+    public Observable<DocumentRelatedResponse> removeDocument(String entityType, int entityId,
```

Comment:
@tarun0 When we know what will be response for many requests the It's better to follow genericity and Please use `GenericResponse`